pkg/userdata/ubuntu/testdata/cri-o-1.9-digitalocean.golden

```diff
@@ -78,29 +78,9 @@ write_files:
       done
       exit 1
     fi
-- path: "/usr/local/bin/download-crictl"
-  permissions: '0777'
-  content: |
-    #!/bin/bash
-    set -xeuo pipefail
-    mkdir -p /opt/bin /opt/cni/bin /etc/cni/net.d /var/run/kubernetes /var/lib/kubelet /etc/kubernetes/manifests /var/log/containers
-    if ! [[ -x /opt/bin/crictl ]]; then
-      for try in {1..3}; do
-        curl -L --fail https://github.com/kubernetes-incubator/cri-tools/releases/download/v0.2/crictl-v0.2-linux-amd64.tar.gz |tar -xzC /opt/bin
-        break
-      done
-    fi
-
-- path: "/etc/systemd/system/crictl-binary.service"
-  content: |
-    [Unit]
-    Requires=network-online.target
-    After=network-online.target
-
-    [Service]
-    Type=oneshot
-    RemainAfterExit=true
-    ExecStart=/usr/local/bin/download-crictl
+- path: /etc/sysctl.d/10-ipv4-forward.conf
```

Comment:
oh, that's why we need to fire `sysctl --system` ?

Reply:
Yup

Reply:
why do we want to enable IP forwarding then :) ?

Reply:
Because it is required by Kubernetes. Docker does enable it by default which is why it is only required for cri-o. The check was added in kubeadm 1.11 but it is required to be enabled on earlier versions as well: https://github.com/kubernetes/kubernetes/pull/63872